src/status.md

```diff
@@ -0,0 +1,313 @@
+<No/> Node.js &nbsp;
+<Wa/> Wasmtime (<sup>1</sup> native i64 support)
+
+## Language features
+
+As such, certain higher-level language features still have their limitations or are not yet available. From a language perspective:
+
+| Feature                                           | What to expect?
+|---------------------------------------------------|-----------------
+| 🐤 **Functional**
+| [Bootstrap](#bootstrap)                           | The compiler can compile itself to WebAssembly, passing the test suite.
+| [Classes and interfaces](#classes-and-interfaces) | Largely implemented in linear memory. Some caveats. (needs GC 🦄)
+| [Standard library](#standard-library)             | Largely implemented in linear memory. Some caveats.
+| [Generics](#generics)                             | Monomorphized templates for now. (maybe post-MVP GC 🦄)
+| [Garbage collection](#garbage-collection)         | Implemented in linear memory for now. (needs GC 🦄)
+| [Interop with JS](#interop-with-js)               | Enabled by the loader package. (needs Type imports / Interface Types 🦄)
+||
+| 🐣 **Limited**
+| [Union types](#union-types)                       | Nullable class types only. Can use generics with static type checks instead. (No proposal so far)
+| [Symbols](#symbols)                               | Implemented, but no deep compiler integration yet.
+| [JSON](#json)                                     | Third-party library available.
+| [RegExp](#regexp)                                 | Third-party library available.
+||
+| 🥚 **Not implemented**
+| [Closures](#closures)                             | Perhaps implement in linear memory. (needs Function references 🦄)
+| [Iterators](#iterators)                           | Not implemented yet. Depends on symbols.
+| [Rest parameters](#rest-parameters)               | Perhaps implement in linear memory. (No proposal so far)
+| [Exceptions](#exceptions)                         | Throwing currently aborts the program. (needs Exception handling 🦄)
+| [Promises](#promises)                             | There is no concept of async/await yet due to the lack of an event loop. (No proposal so far)
+| [BigInt](#bigint)                                 | There are no BigInts yet, but there are i64s.
+||
+| 🕳️ **Not supported**
+| [Dynamicness](#dynamicness)                       | AssemblyScript avoids overly dynamic JavaScript features by design.
+
+### Bootstrap
+
+The first release able to bootstrap itself and pass the test suite is v0.18, released in January 2021. Note that the compiler is not technically "self hosted" in WebAssembly still, as it currently uses a JavaScript frontend for I/O and links to Binaryen (C++ with Emscripten), which also requires some JavaScript glue code. As such, to make the compiler work in a WebAssembly-only engine like Wasmtime, the next steps would be to work towards a WebAssembly-only build of Binaryen, and replace the I/O parts provided by `asc` with WASI.
```

Comment:
Excited for this! :smile: :tada: 

Another alternative too, would be to use node's WASI implementation for ease of use in npm. But Wasmtime would be the coolest one to use for sure :smile:

Reply:
One challenge is that we may also want to keep providing browser builds, so we may end up with two abstractions there eventually. The first makes everything use WASI so we can run the compiler with node's or Wasmtime's WASI implementation, and the browser one would translate WASI calls back to something a browser understands, heh :)